### ISSUE TITLE
Really loved the design of this portfolio. Have some queries about its inspiration.

### ISSUE THREAD

**Issue by KevinKhachariya** [Suhel-Kap/portfolio#3]
Hello there @Suhel-Kap !

Whenever I am thinking different ways to design my portfolio design, I always come back to admire the way you designed yours.

So just wanted to ask where did you got the inspiration for design. Especially the Home and Experience section looks clean easy to understand and modern.

Just curious so would also ask if you ever considered talking help of Artificial Intelligence's help for any of the development process.

**Comment by Suhel-Kap** [comment]
Hey @KevinKhachariya 

Really appreciate the kind words.

To be honest I am not a UI developer, so it took me a lot of research to go through a ton of portfolio website templates and choose the best things from them. You can look for the innumerous templates on GitHub & Figma to take inspiration. 
If you want you can also use my website by tweaking the colors a bit to your liking. The README file has steps mentioned to get it working for you.

I created this website more than a year ago, so i didn't use any aid of an AI, but with the advent of ChatGPT, you can definitely make it a lot better. ✌

**Comment by KevinKhachariya** [comment]
Designing these UI is definitely a lot difficult than it looks. I too experienced the need of extensive research to find something fitting. Your design has true modern aesthetics. Really good repository to take inspiration from. Well not only design but UX is properly considered. Definitely way better job than I did on a project when I finally tried to be in the same shoes as UI devs.

If I had to suggest to improve it even further then it would be these two tools 1. [Markup validator](https://validator.w3.org/) 2. [web accessibility](https://wave.webaim.org/) which level ups your website to welcome all users truly. I would suggest you to check it out for accessibilties. I know profolio are not for wide range of visitors so a perfect time to just evaluate different ways to improve existing websites.
